src/vehicle_use.cpp

```diff
@@ -1092,6 +1093,38 @@ void vehicle::play_chimes()
     }
 }
 
+void vehicle::crash_terrain_around() {
+    const tripoint around[] = { {1, 1, 0}, { 0,1,0 }, {-1, 1, 0}, { 1, 0, 0}, {0, 0, 0}, { -1, 0, 0}, {1, -1, 0}, {0, -1, 0}, {-1, -1, 0} };
```

Comment:
We have a collection of neighboring tripoints in https://github.com/CleverRaven/Cataclysm-DDA/blob/ec26a7eaacc6fb391fb9f341314c4ce1b2b9af8e/src/point.h#L352

Reply:
ok. Thanks, I've fixed it.

Reply:
Failing test is unrelated to your changes it probably caught #33640

Reply:
and how I can fix it?

Reply:
ok. I've merged my branch with the newest changes. Maybe, it could help.